.github/workflows/go.yaml

```
@@ -54,6 +54,33 @@ jobs:
 
       - name: Linting & vetting.
         run: make go-lint
+  proto:
+    runs-on: ubuntu-latest
+    name: Check protobuf generated code
+    env:
+      GOBIN: /tmp/.bin
+    steps:
+      - name: Checkout code into the Go module directory.
+        uses: actions/checkout@v2
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Generate protobuf code
+        run: make proto
+
+      - name: Format
+        run: make format
+
+      - name: Format
```

```suggestion
      - name: Check proto
```

Nit: Something like this?